packages/shared/src/guards.ts

```diff
@@ -142,14 +142,6 @@ export function isTypedArray(val: unknown): val is TypedArray {
   );
 }
 
-export function assertArrayOrTypedArray(
```

Comment:
This should be really be called `assertValueArray`, but since it's used only below, I chose to inline it.